app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.java

```diff
@@ -446,7 +444,7 @@ private void scrollToCurrent(@NonNull List<Lecture> lectures, int day, int heigh
 
             for (Lecture l : lectures) {
                 if (l.day == day && l.startTime <= time && l.startTime + l.duration > time) {
-                    if (col == -1 || col >= 0 && l.roomIndex == MyApp.roomList.get(col)) {
+                    if (col == -1 || col >= 0 && l.roomIndex == col) {
```

Comment:
Kind of, but not really. It could have been done as soon as `ScheduleData` was introduced, because that changed the room index to colum index mapping to be an identity mapping (1 -> 1, 2 -> 2, etc). But `MyApp.roomList` only existed for this reason. So after you replace the lookups `MyApp.roomList` is no longer used anywhere.

Reply:
Please summarize the fact in the commit message as it might not be obvious to the reader. Also please reference the commit which introduced the identity mapping to provide context.

Reply:
Resolved in be0a58b634e65950a8777e8e8eab898a0a5fe9cf. Merci.